### PR TITLE
Style updates for featured content blocks on tag pages

### LINKED ISSUE #441: Fix header padding on featured content block for tag pages

### PR DESCRIPTION
**Pull request by katjuell**
### What should this PR do?
Resolves #441 by adjusting padding on header text for featured content blocks on tag pages.

Also takes another look at the `max-height` that got set in #417 and fixes linting (which was broken, due to renovate's auto updating of `eslint-plugin-unicorn`).

### Why are we making this change?
- We want styles to look good.

### What are the acceptance criteria? 
- Tag content blocks look correct in all browsers, old and new.

### How should this PR be tested?
- Check out the branch, and install deps with `npm ci`
- Check that linting works.
- Places to check: tag pages (`/tags/<whatever>`), homepage, index pages (`/posts`, `/videos`).
- Check out the deploy url in major browsers (Chrome, Firefox, Safari), as well as an older Safari, and check the above pages.

## Pull request process

**Reviewers**:

1. Test functionality using the criteria above.
2. Offer tips for efficiency, feedback on best practices, and possible alternative approaches and things that may not have been considered.
3. For shorter, "quick" PRs, use your best judgement on #​2.
4. Use a collaborative approach and provide resources and/or context where appropriate.
5. Provide screenshots/grabs where appropriate to show findings during review.

**Reviewees**:

1. Prefer incremental and appropriately-scoped changes.
2. Leave a comment on things you want explicit feedback on.
3. Respond clearly to comments and questions.

### CODE REVIEW

**Comment by katjuell** [comment]
Ideal state:
Screen Shot 2022-01-03 at 5 06 27 PM

File: components/atoms/CardLink/CardLinkStyles.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ interface Props {
 export const StyledCardLink = styled.a<Props>`
     cursor: pointer;
     height: 100%;
-    max-height: 377px;
+    max-height: 450px;
```
Comment (bretthayes) [inline components/atoms/CardLink/CardLinkStyles.ts:10]: This seems restrictive and may cause problems when content is stacked or flows naturally with responsiveness. I'm seeing issues on and below Safari 15.1 on Monterey.

The card margins are ignored and the link element wrapping the card has `margin-top: 0;`.

Could we add `max-height: 100%;` instead to the card? It seems to fix the issue.

bs_macmty_Safari_15 1

**Review by bretthayes — CHANGES_REQUESTED** [review]

File: components/atoms/CardLink/CardLinkStyles.ts (hunk 1 above)
Comment (katjuell) [reply components/atoms/CardLink/CardLinkStyles.ts:10]: Sounds good to me!

File: components/atoms/CardLink/CardLinkStyles.ts [hunk 2]
```diff
@@ -7,7 +7,7 @@ interface Props {
 export const StyledCardLink = styled.a<Props>`
     cursor: pointer;
     height: 100%;
-    max-height: 377px;
+    max-height: 100%;
```
Comment (bretthayes) [inline components/atoms/CardLink/CardLinkStyles.ts:10]: This seems to work down to Safari 11.1. ✅

The cards start to break on and below 10.1. ❌ I'm not sure how far we want to support but I think we should try and support as far back as possible. Here are some screenshots:

Screen Shot 2022-01-04 at 3 28 52 PM

Screen Shot 2022-01-04 at 3 29 28 PM

**Review by bretthayes — APPROVED** [review]
Approving for now since this is an improvement compared to what we currently have in production. We should create a follow up ticket to improve and support older browsers with more cross-browser styling support.

**Comment by katjuell** [comment]
That sounds good to me!

**Pull request merged**